server/fleet/queries.go

```diff
@@ -196,9 +206,10 @@ func (tq *TargetedQuery) AuthzType() string {
 }
 
 var (
-	errQueryEmptyName  = errors.New("query name cannot be empty")
-	errQueryEmptyQuery = errors.New("query's SQL query cannot be empty")
-	errInvalidLogging  = fmt.Errorf("invalid logging value, must be one of '%s', '%s', '%s'", LoggingSnapshot, LoggingDifferential, LoggingDifferentialIgnoreRemovals)
+	errQueryEmptyName       = errors.New("query name cannot be empty")
+	errQueryEmptyQuery      = errors.New("query's SQL query cannot be empty")
+	ErrQueryInvalidPlatform = errors.New("query's platform must be a comma-separated list of 'darwin', 'linux', 'windows', and/or 'chrome' in a single string")
```

Comment:
just curious why this is the only exported error, maybe the others aren't tested outside the package?

Reply:
@mostlikelee exactly – testing for precisely this error in the datastore layer test